src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java

```diff
@@ -39,24 +38,30 @@ public void scheduleFixedDelayTask() {
     MDC.put(LoggingPlaceholders.TRACE_ID_PLACEHOLDER,
         "- [CID= " + UUID.randomUUID().toString().toLowerCase() + "]");
     logger.logSchedulerStart();
-    Integer offset = 0;
-    while (true) {
-      PollingEserviceResponse response = eserviceService.getEservicesReadyForPolling(limit, offset);
-      for (EserviceContent service : response.getContent()) {
-        try {
-          servicesSend.sendMessage(service);
-          eserviceService.updateLastRequest(service.getEserviceRecordId(),
-              ChangeLastRequest.builder().lastRequest(OffsetDateTime.now(ZoneOffset.UTC)).build());
-        } catch (IOException e) {
-          logger.logQueueSendError(service.getEserviceRecordId());
+    try {
+      Integer offset = 0;
+      while (true) {
```

Comment:
Use Async development with Threads

Reply:
we already have IESP-140 task for this in our backlog